src/BuildScriptGenerator/DotNetCore/DotNetCoreBashBuildSnippet.sh.tpl

```diff
@@ -16,8 +16,18 @@ if [ "$SOURCE_DIR" == "$DESTINATION_DIR" ]
 then
     dotnet publish "{{ ProjectFile }}" -c {{ Configuration }}
 else
-    echo
-    echo "Publishing to directory $DESTINATION_DIR..."
-    echo
-    dotnet publish "{{ ProjectFile }}" -c {{ Configuration }} -o "$DESTINATION_DIR"
+	echo
+	echo "Publishing to directory $DESTINATION_DIR..."
+	echo
+	dotnet publish "{{ ProjectFile }}" -c {{ Configuration }} -o "$DESTINATION_DIR"
+
+	# we copy *.csproj to destination directory so the detector can identify
+	# the destination directory as a DotNet application
+	# when running oryx run-script
+	# 
+	# 2>/dev/null || :
+	# code snippet above is used to surpass cp error message & code
+	# since this is needed during: oryx run-script 
+	# but not during other dotnet builds
+	cp ${SOURCE_DIR}/*.csproj ${DESTINATION_DIR} 2>/dev/null || :
```

Comment:
nit: extra tab/spacing at the beginning of this section?

Reply:
Updated. Thanks!